test/Microsoft.AspNetCore.Server.Kestrel.Core.Tests/MessageBodyTests.cs

```diff
@@ -28,18 +29,18 @@ public class MessageBodyTests
             using (var input = new TestInput())
             {
                 var body = MessageBody.For(httpVersion, new FrameRequestHeaders { HeaderContentLength = "5" }, input.Frame);
-                var stream = new FrameRequestStream();
+                var stream = new FrameRequestStream(Mock.Of<IHttpBodyControlFeature>());
                 stream.StartAcceptingReads(body);
 
                 input.Add("Hello");
 
                 var buffer = new byte[1024];
 
-                var count = stream.Read(buffer, 0, buffer.Length);
+                var count = await stream.ReadAsync(buffer, 0, buffer.Length);
```

Comment:
This test (and similar-named ones) are specifically testing sync reads. Pass a mock feature that allows sync I/O.